c_glib/plasma-glib/client.cpp

```diff
@@ -431,6 +434,29 @@ gplasma_client_refer_object(GPlasmaClient *client,
   }
 }
 
+/**
+ * gplasma_client_disconnect:
+ * @client: A #GPlasmaClient.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: %TRUE on success, %FALSE if there was an error.
+ *
+ * Since: 0.12.0
+ */
+gboolean
+gplasma_client_disconnect(GPlasmaClient *client,
+                          GError **error)
+{
+  auto priv = GPLASMA_CLIENT_GET_PRIVATE(client);
+  auto status = priv->client->Disconnect();
+  if (garrow_error_check(error, status, "[plasma][client][disconnect]")) {
+    priv->disconnected = true;
```

Comment:
We should use `bool` instead of `gboolean` when we use `true`.

Reply:
I see. I'll use `bool`.